app/src/views/features/sessions/SessionsIndexPageContainer/SessionsIndexPage/OnboardingView.tsx

```diff
@@ -155,11 +171,143 @@ const SessionOnboardingView: React.FC<SessionOnboardProps> = ({ redirectToSettin
+            <Text type="secondary" className="banner-text">
+              <div>
+                Safely capture <Text strong>mouse movement</Text>, <Text strong>console</Text>,{" "}
+                <Text strong>network</Text> &
+              </div>
+              <div>
+                {" "}
+                <Text strong>environment data</Text> automatically on your device for sharing &{" "}
+              </div>
+              <div> debugging </div>
+            </Text>
+          </Row>
+          <Row className="record-label">
+            <Text type="secondary" className="banner-text">
+              Record your first session
+            </Text>
+          </Row>
+          <Row>
+            <Col span={15} className="input-container">
+              <Input ref={inputRef} placeholder="Enter the URL you want to record" />
+            </Col>
+            <Col span={3} className="start-btn-container">
+              <Button size="middle" type="primary" onClick={handleStartRecordingBtnClicked}>
+                {" "}
+                Start Recording
+              </Button>
+            </Col>
+          </Row>
+        </Col>
+        <Col span={13} className="banner-demo-video">
+          <Row justify="end">
+            <img src={StartSessionRecordingGif} alt="How to start session recording" className="demo-video" />
+          </Row>
+          <Row onClick={trackOnboardingYTVideoClicked}>
+            <a href="https://www.youtube.com/embed/g_qXQAzUQgU?start=74" target="__blank" className="yt-cta-container">
+              <Row justify="end" align="middle" className="yt-cta">
+                <YoutubeFilled style={{ color: "red", fontSize: 18, marginTop: 4, margin: 0 }} /> &nbsp;
+                <Text underline>Watch it in action</Text>
+              </Row>
+            </a>
+          </Row>
+        </Col>
+      </Row>
+      <div className="onboarding-footer">
+        <div className="divider">
+          <div>
+            <Divider />
+          </div>
+        </div>
+        <Row>
+          <Col span={24}>
+            <Text type="secondary">
+              <Row justify="space-evenly">
+                <CheckItem label="Automatically record specified website or all activity" />
+                <CheckItem label="All recordings are saved locally until saved or shared" />
+                <CheckItem label="View network, console and environment details synced to video" />
+              </Row>
+            </Text>
+          </Col>
+        </Row>
+      </div>
+      <InstallExtensionModal
+        open={isInstallExtensionModalVisible}
+        onCancel={closeModal}
+        heading="Install Browser extension to record sessions for faster debugging and bug reporting"
+        subHeading="Safely capture mouse movement, console, network & environment data automatically on your device for sharing and debugging. Private and secure, works locally on your browser."
+        eventPage="session_recording_page"
+      />
```

Comment:
We have this modal in redux. We can just toggle this using `dispatch(actions.toggleActiveModal({ modalName: "extensionModal",...props}));`